tensorflow/contrib/stateless/python/stateless_ops.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Since you're creating a new file how about adding it to tensorflow/python/ops already instead of contrib?

(also I think it's copyright 2018 not 2015)

Reply:
Date fixed, but it seems cleaner to have one commit that rewrites `tf.contrib.stateless` and another that moves it out of contrib.  Would you prefer I do it all at once?  Also, whenever I move it: should I leave `tf.contrib.stateless` around or delete it entirely?